src/pages/test-report/test-report.effects.ts

```diff
@@ -107,10 +107,10 @@ export class TestReportEffects {
     withLatestFrom(
       this.store$.pipe(
         select(getTests),
-        map(isPracticeTest),
+        map(isTestReportPracticeTest),
       ),
     ),
-    filter(([action, isPracticeTest]) => !isPracticeTest),
+    filter(([action, isTestReportPracticeTest]) => !isTestReportPracticeTest),
```

Comment:
Does this mean we are going to need a double filter everywhere we want to not do something if it's practice mode (or double up on if statements) , seems like we would want to keep an isPracticeMode selector which can tell us this instead of separating them out

Reply:
We have to identify the different practice modes so that we know when to exit each of them. However on the presentational layer we will have only one variable which indicates if it's practice mode not, but we don't need to know which.